app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/TimeStampFeatureAttribute.ts

```diff
@@ -0,0 +1,13 @@
+import {FeatureAttributeName} from "../states/map/FeatureAttributeName";
+import {FilterName} from "../states/map/FilterName";
+import {FeatureAttribute} from "./FeatureAttribute";
+
+export class TimestampFeatureAttribute extends FeatureAttribute {
+  static readonly instance: TimestampFeatureAttribute = new TimestampFeatureAttribute();
```

Comment:
You also don't need the : TimestampFeatureAttribute if it's obvious from the right hand side, which it is.